### PR TITLE
[eas-attestation] EAS Attestation Proposal Validation

### PR DESCRIPTION
**Pull request by andremury**
Changes proposed in this pull request:
- [X] New validation schema using EAS's Attestation. 
 
Use [Attest.sh](http://attest.sh) to allow users to create a new proposal in the DAO's space

#### Tests: passing
image

### CODE REVIEW

File: src/validations/eas-attestation/index.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import Validation from '../validation';
+import fetch from 'cross-fetch';
+
+export const author = 'KarmaHQ';
```
Comment (ChaituVR) [inline src/validations/eas-attestation/index.ts:4]: Hi @andremury PR should be comming from KarmaHQ github fork
```suggestion
export const author = 'andremury';
```

File: src/validations/eas-attestation/index.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+export const version = '0.1.1';
```
Comment (ChaituVR) [inline src/validations/eas-attestation/index.ts:5]:
```suggestion
export const version = '0.1.0';
```

File: src/validations/eas-attestation/index.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  public id = 'eas-attestation';
+  public github = 'karmahq';
+  public version = '0.1.0';
+  public title = 'EAS Attestation';
+  public description =
+    'Use EAS attest.sh to determine if user can create a proposal.';
+
+  async validate(): Promise<boolean> {
+    if (this.params.strategies?.length > 8)
+      throw new Error(`Max number of strategies exceeded`);
```
Comment (ChaituVR) [inline src/validations/eas-attestation/index.ts:76]: Looks like you are not using this param anywhere. 🤔

File: src/validations/eas-attestation/index.ts (hunk 1 above)
Comment (andremury) [reply src/validations/eas-attestation/index.ts:4]: Hey, thanks for the feedback. I'm gonna create a PR from there

**Comment by andremury** [comment]
Replaced by #1238

**Pull request closed without merging**